dev/TreeView/TreeView.idl

```diff
@@ -94,6 +94,14 @@ runtimeclass TreeViewDragItemsCompletedEventArgs
     }
 }
 
+[WUXC_VERSION_RS5]
```

Comment:
Was there a specific reason you chose WUXC_VERSION_RS5?
If our event is available in RS4, than so should our Eventargs be.

Reply:
No, reason. This is good feedback, I changed to `WUXC_VERSION_RS4`.